dart/lib/src/isolate_error_integration.dart

```diff
@@ -5,15 +5,17 @@ import 'protocol.dart';
 import 'sentry_options.dart';
 import 'throwable_mechanism.dart';
 
-/// integration that capture errors on the current Isolate Error handler
-/// which is the main thread.
-void isolateErrorIntegration(Hub hub, SentryOptions options) {
-  final receivePort = _createPort(hub, options);
+Integration initIsolateErrorIntegration(
```

Comment:
on Android we implement `Closeable` https://github.com/getsentry/sentry-java/blob/main/sentry-android-core/src/main/java/io/sentry/android/core/TempSensorBreadcrumbsIntegration.java#L23
which is the equivalent of this technique for `typedef` I'd say, sounds good.

Reply:
we don't use `init` as a prefix for the other integrations, I'd rather stick to the normal naming

Reply:
✅